educator/src/Dscp/DB/SQLite/Queries.hs

```diff
@@ -271,20 +271,21 @@ getStudentTransactions student = do
 
 data GetProvenStudentTransactionsFilters = GetProvenStudentTransactionsFilters
     { pfCourse :: Maybe Course
+    , pfStudent :: Maybe Student
+    , pfAssignment :: Maybe (Hash Assignment)
     , pfSince  :: Maybe UTCTime
     } deriving (Show, Generic)
 
 instance Default GetProvenStudentTransactionsFilters where
-    def = GetProvenStudentTransactionsFilters def def
+    def = GetProvenStudentTransactionsFilters def def def def
```

Comment:
Why not
```haskell
deriving instance Default
    GetProvenStudentTransactionsFilters
```
?

Reply:
Loooooool, magically!

Reply:
Because didn't care enough at the moment.
Also, requires enabling `DeriveAnyClass`, what's our policy on that?
Pushed anyway.